docs/transcricoes.md

```diff
@@ -1,10 +1,10 @@
 # Transcrições
 
 ## Distinção entre Tradução e Transcrição
-Transcrever é o ato de redigir um texto que assimile a fala do palestrante, tal como ele a reproduz. É importante que contenha, com a máxima fidelidade possível, as palavras deste.
+Transcrever é o ato de redigir um texto que assimile a fala do palestrante, tal como ele a reproduz.(Eu devo ter escrito isso, mas agora estou achando um pouqo estranho esse "tal como ele a reproduz". Reproduzir dá a ideia de "produzir novamente" "copiar" etc.) É importante que contenha, com a máxima fidelidade possível, as palavras deste.
 Assim, para evitar descaracterizar a fala dos palestrantes que utilizam uma linguagem mais coloquial, serão aceitas algumas formas gramaticais que, apesar de não serem abonadas num texto formal culto, são adequadas e próprias num contexto mais informal.
 
-Tenha o devido cuidado de evitar sílabas vazias, repetições ou tropeços do palestrante. Caso haja um erro óbvio, pode-se corrigi-lo, desde que a correção esteja entre colchetes, para indicar que não foi o que ele disse. Também é aceitável reescrever o que o palestrante disse em casos onde se precise adequar a legenda para que tenha velocidade de leitura e comprimento adequados. A transcrição de legendas também deve incluir informações sonoras como aplausos e risos, para os deficientes auditivos, tal como a sincronia com a fala do palestrante. Traduções ocorrem quando a palestra foi ministrada em um idioma estrangeiro que não seja o Português Brasileiro, caso em que utilizamos uma transcrição do idioma original como texto base para a criação da legenda traduzida. No fim, ambas são portadoras de ideias, transcreva para transmiti-las e traduza para recebê-las :).
+Tenha o devido cuidado de evitar sílabas vazias, repetições ou tropeços do palestrante. Caso haja um erro óbvio, pode-se corrigi-lo, desde que a correção esteja entre colchetes, para indicar que não foi o que ele disse. Também é aceitável reescrever o que o palestrante disse em casos onde (em que) se precise adequar a legenda para que tenha velocidade de leitura e comprimento adequados. A transcrição de legendas também deve incluir informações sonoras como aplausos e risos, para os deficientes auditivos, tal como a sincronia com a fala do palestrante. Traduções ocorrem quando a palestra foi ministrada em um idioma estrangeiro que não seja o Português Brasileiro (acho que as iniciais aqui devam ser minúsculas: português brasileiro) (também estou pensando nesta frase: ministrada em um idioma estrangeiro que não seja o português brasileiro: o português não é idioma estrangeiro, pelo menos para nós), caso em que utilizamos uma transcrição do idioma original como texto base para a criação da legenda traduzida. No fim, ambas são portadoras de ideias, transcreva para transmiti-las e traduza para recebê-las :).
```

Comment:
A mesma coisa que disse no outro, altere o que for preciso e faça os comentários fora do arquivo.